canu/generate/switch/config/config.py

```diff
@@ -1097,6 +1103,8 @@ def groupby_vlan_range(vlan_list):
     Returns:
         list of vlans formatted
     """
+    if None in vlan_list:
+        return
     if not len(vlan_list):
```

Comment:
I know this is an older bit of code, but this seems like a weird test result.  The doc says we return a (sorted) list of vlans, but here we return a singular string?  Probably need to make sure we're returning things consistently: a list, a string, something.